spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/job/support/StepExecutionProgressInfo.java

```diff
@@ -64,7 +64,7 @@ public StepExecutionProgressInfo(StepExecution stepExecution, StepExecutionHisto
 		if (startTime == null) {
 			startTime = LocalDateTime.now();
 		}
-		duration = Duration.between(startTime, endTime).get(ChronoUnit.MILLIS);
+		duration = Duration.between(startTime, endTime).get(ChronoUnit.NANOS);
```

Comment:
Why the switch from millis to nanos?

Reply:
I can not remember why this was switched to NANOS.   Will revert.